server/views/pages/visit/summary.njk

```diff
@@ -67,22 +68,63 @@
         {% set cancelButtonClasses = 'govuk-!-margin-top-5' %}
         {% if updateJourneyEnabled %}
           {% set cancelButtonClasses = 'govuk-!-margin-top-5 govuk-button--secondary' %}
+
+          {% if visit.visitStatus === 'BOOKED' %}
+            {{ govukButton({
+              text: "Update booking",
+              classes: "govuk-!-margin-top-5",
+              href: "/visit/" + visit.reference + "/update/select-visitors",
+              attributes: { "data-test": "update-visit" },
+              preventDoubleClick: true
+            }) }}
+          {% else %}
+            {{ govukButton({
+              text: "Update booking",
+              classes: "govuk-!-margin-top-5",
+              attributes: { "data-test": "update-visit" },
+              preventDoubleClick: true,
+              disabled: true
+            }) }}
+          {% endif %}
+        {% endif %}
+
+        {% if visit.visitStatus === 'BOOKED' %}
```

Comment:
Again I believe you can do the same here unless it's doing something odd when the button has a `href` and isn't disabling it?